specs/Badge.md

```diff
@@ -0,0 +1,118 @@
+# [Badge](https://github.com/microsoft/fluentui/issues/16925)
+
+## Background
+
+A badge is an additional visual discriptor for UI elements.It can be used to denote numerical value, status or general information.
+
+## Prior Art
+
+- [OpenUI Research](https://open-ui.org/components/badge.research)
+
+## Sample Code
+
+```jsx
+  <Badge>
+    My Custom Badge
+  </Badge>
+  <Badge
+    style={{ position: 'absolute', top: -4, right: -4 }}
+  />
+```
+
+## Variants
+
+`Badge` can have several variations.
+
+- Appearance: `default`, `rounded` and `circular`
+- Size: `smallest`, `smaller`, `small`, `medium`, `large`, `larger`.
+- Styles: `filled`, `outline`, `ghost`, `tint`, `inverted filled`
+
+## API
+
+| Property Name | Type   | Default Value | Description                         |
+| ------------- | ------ | ------------- | ----------------------------------- |
+| `size`        | `enum` | `medium`      | The Badge size                      |
+| `shape`       | `enum` | `rounded`     | The Badge can have different shapes |
+| `variant`     | `enum` | `filled`      | The Badge different style variants  |
```

Comment:
You should include all properties that the `BadgeProps` type will have. It'd probably be best to just include the planned contents of Badge.types.ts instead of this table. That way you can just paste it into the implementation, and don't need to duplicate work writing this table then translating it to code later. Or, since you already started the implementation, just paste the contents of Badge.types.ts here.

(See the [Avatar spec](https://github.com/microsoft/fluentui/blob/master/packages/react-avatar/SPEC.md#API) for an example of what I mean.)